src/main/java/vct/main/stages/Transformation.scala

```diff
@@ -149,14 +149,21 @@ case class SilverTransformation
     EncodeSendRecv,
     ParBlockEncoder,
 
-    // Encode proof helpers
-    EncodeProofHelpers,
-
     // Encode exceptional behaviour (no more continue/break/return/try/throw)
     SpecifyImplicitLabels,
     SwitchToGoto,
     ContinueToBreak,
     EncodeBreakReturn,
+
+    SplitQuantifiers,
+    ) ++ simplifyBeforeRelations ++ Seq(
+    SimplifyQuantifiedRelations,
+    SimplifyNestedQuantifiers,
+    ) ++ simplifyAfterRelations ++ Seq(
+
+    // Encode proof helpers
+    EncodeProofHelpers,
```

Comment:
Encode proof helpers changed FramedProofs requires and ensures to be changed to loop invariants with `once ==> ...` and `!once ==> ..`, which made SimplifyNestedQuantifiers not work anymore, since it did not allow to extract things like `threadBlockSize == 32`. (since we now have `once ==> threadBlockSize == 32`). 

I could change it I suppose to work with that, but it makes the pass more difficult with more bookholding.

Similar for `EncodeTryThrowSignals`. What would you suggest?